tox.ini

```diff
@@ -87,7 +87,7 @@ extras =
 
 commands =
     devdeps: pip install -U -i https://pypi.anaconda.org/astropy/simple astropy --pre
-    devdeps: pip install -U -i https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
+    devdeps: pip install -U -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple scikit-learn
```

Comment:
this is a wee bit premature, though definitely will be done, but atm scikit-learn hasn't yet moved the nightly location (nor pandas).

Reply:
Should we turn this PR to draft? I was using blind `sed` in batch mode. 😅

Reply:
or just leave it open for the time being. 

I was waiting for those two remaining ones to happen before batchpr-ing 😅

Reply:
Ooops sorry for doing this too soon, but I only did it for `astropy` and `spacetelescope` orgs.

Reply:
I already did this a couple hours ago 😅 
https://github.com/astropy/photutils/pull/1549

Reply:
Wut...? You could time travel?!

Reply:
Your bot somehow didn't pull the latest `main` branch before this PR.

Reply:
Sorry, I see this is just for `scikit-learn`.  Yeah, I didn't update this URL yet because I saw it was unavailable.  I was planning to replace this with `scikit-learn>=0.0.dev0` (like the other dev deps), when it was available.

Reply:
Larry is usually extra up-to-date with any changes happening in the ecosystem. Also, the bot was run correctly, not that there are two links in the config file, somehow scikit-learn didn't make it to the group of the other upstream packages.

So eventually an update will be useful, but it might need to be a manual one to move scikit-learn into the main group. (btw, mpl is also haven't fully finished the migration, there have been some token stuff to work out as well as waiting for dependencies to migrate first, etc, so overall I would give it a few more days for the dust to settle for all the packages)

Reply:
Okay, I like Larry's plan better. Feel free to close this!